light-client/relayer/src/error.rs

```diff
@@ -0,0 +1,26 @@
+use displaydoc::Display;
+use mc_watcher::error::WatcherDBError;
+
+#[derive(Debug, Display)]
+pub enum Error {
+    /// Thread join error
+    ThreadJoin,
+
+    /// Burned transaction verification error
+    InvalidBurnedTx(mc_light_client_verifier::Error),
+
+    /// Unable to get block signatures from watcher
```

Comment:
```suggestion
    /// WatcherDb: {0}
```

`Unable to get block signatures from watcher` is only true if we are doing a very specific operation against the watcher db